.github/workflows/build-and-publish-base.yaml

```diff
@@ -6,15 +6,16 @@ on:
       bitops_base_tag:
         description: Specifies the tag that will be published
         type: string
-        default: plugins-base
+        default: base
         required: true
       bump_base_tag:
         description: Bumps the bitops-tag.yaml file, this will trigger the recreation of the official prebuilt plugins images
         type: boolean
         default: false
 
   push:
-    branches: [ main ]
+    # TODO: Remove debugging
+    branches: [ fix/base-docker-tagging ]
```

Comment:
I would expect the push branches to be main.

Reply:
Right.
It's reverted after confirming the working functionality with testing.